comit/src/rsa.rs

```diff
@@ -0,0 +1,115 @@
+//! Recommended Safe Actions
+//!
+//! This module provides logic for recommending when it is safe/unsafe to take a
+//! particular action for the COMIT protocol.
+//!
+//! 'Safe' is defined as:
+//! 1 - You will not loose money.
+//! 2 - The counterparty is likely to continue with the swap because it is safe
+//!     for them to do so.
+
+use crate::Timestamp;
+use time::Duration;
```

Comment:
Why is `std::time::Duration` not good enough?

Reply:
This type will be hold time on a minute granularity, `std::time::Duration` seems designed for smaller periods, specifically the docs say:
"
A Duration type to represent a span of time, typically used for system timeouts.
"

This will be blocktimes, and 'time to act' (possibly even measured in hours).